irohad/consensus/yac/impl/timer_impl.hpp

```diff
@@ -21,6 +21,7 @@
 #include <rxcpp/rx.hpp>
 #include "consensus/yac/timer.hpp"
 
+// TODO: This timer should not be here under YAC, but should be some generic util that YAC uses.
```

Comment:
The idea was that YAC consumes `Timer` interface, which provides some methods. It happened that we require different types of timers in modules -- `OrderingService` should also perform actions based on network notificatons, together with timer. So this implementation is local to YAC.